### PR TITLE
Composite run action to deploy to cloud.gov

### PR DESCRIPTION
**Pull request by rahearn**

### CODE REVIEW

File: .github/workflows/action.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+name: "test"
```
Comment (Sgtpluck) [inline .github/workflows/action.yml]: [nit] "cloud.gov deploy"?

File: action.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+name: "cloud.gov deploy"
```
Comment (Sgtpluck) [inline action.yml:1]: [question] how does this file work with the `/github/workflows/actions.yml` file?

File: .github/workflows/action.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+on:
+  pull_request:
```
Comment (Sgtpluck) [inline .github/workflows/action.yml]: [question] is this a holdover from a past action? i'm guessing we don't want the deploy action running on `pull_request`?

**Review by Sgtpluck — CHANGES_REQUESTED** [review]
I'd love a quick review chat where you can make it a little clearer how the two `action.yml` files are working together?

File: action.yml (hunk 2 above)
Comment (rahearn) [reply action.yml:1]: This file is the shared action that will be `uses`ed by other workflows. `.github/workflows/action.yml` is the local workflow that tests the shared one.

File: .github/workflows/action.yml (hunk 3 above)
Comment (rahearn) [reply .github/workflows/action.yml]: It's a bad way to do a real deploy, but here it is making sure that the action works on a PR to this repo. This file is not included when other repos use this shared action.

**Pull request merged**